onnxruntime/core/providers/cpu/nn/string_split.cc

```diff
@@ -0,0 +1,112 @@
+#include "string_split.h"
+#include "core/common/common.h"
+#include <algorithm>
+namespace onnxruntime {
+
+ONNX_CPU_OPERATOR_KERNEL(
+    StringSplit,
+    20,
+    KernelDefBuilder()
+        .TypeConstraint("T1", DataTypeImpl::GetTensorType<std::string>())
+        .TypeConstraint("T2", DataTypeImpl::GetTensorType<std::string>())
+        .TypeConstraint("T3", DataTypeImpl::GetTensorType<int64_t>()),
+    StringSplit);
+
+int64_t countSubstrings(std::string_view str, std::string_view substr) {
+  if (str.empty()) {
```

Comment:
It would be helpful to have a short doc as to what exactly this function returns.